task/tkn-bundle/0.1/tkn-bundle.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "image-build, appstudio, hacbs"
+  name: tkn-bundle
+spec:
+  description: |-
+    Creates and pushes a Tekton bundle containing the specified Tekton YAML files.
+  params:
+  - description: Reference of the image task will produce.
+    name: IMAGE
+    type: string
+  - default: .
+    description: Path to the directory to use as context.
+    name: CONTEXT
+    type: string
+  results:
+  - description: Digest of the image just built
+    name: IMAGE_DIGEST
+  - description: Image repository where the built image was pushed
+    name: IMAGE_URL
+  steps:
+  - image: quay.io/redhat-appstudio/appstudio-utils:4580b3ba3012095ff3981e50b6bbf753d4afd4c3
```

Comment:
You probably want this to be just `appstudio-utils`. That way, build-and-push.sh does the right thing [here](https://github.com/redhat-appstudio/build-definitions/blob/258144d6b15f193eb8a007551eef018d7c62afd8/hack/build-and-push.sh#L83).

I also noticed that another task, configure-build, also hard-codes an appstudio-utils image reference directly. Not sure if that's an oversight or intentional.

Reply:
I am trying to remove dependency on appstudio-utils, so that it would not be necessary to built it.